src/broker/controller-dbus.c

```diff
@@ -505,28 +505,59 @@ static int controller_dispatch_object(Controller *controller, uint32_t serial, c
         return CONTROLLER_E_UNEXPECTED_PATH;
 }
 
+static int controller_dispatch_reply(Controller *controller, uint32_t serial, const char *signature, Message *message) {
+        ControllerReload *reload;
+        int r;
+
```

Comment:
Shouldn't we handle errors as well? If the launcher fails, it should be able to reply with an error? This call doesn't require it, but others might in the future. So once we generalize this, we should handle METHOD_RETURN and ERROR in some more generic sense. Fine for now, I think.

Reply:
Correct. Kept it specific for now as the reply objects are anyway not generic.